### ISSUE TITLE
SyntaxError: Named export 'LUTCubeLoader' not found. The requested module 'postprocessing' is a CommonJS module

### ISSUE THREAD

**Issue by mhmdnzr** [pmndrs/postprocessing#490]
### Description of the bug

I use postprocessing like:

```js
import { LUTCubeLoader } from "postprocessing";
```

when i run my nextjs app, it show me an error with this topic: 

`
import pkg from 'postprocessing';
const { EffectComposer: EffectComposer$1, RenderPass, NormalPass, DepthDownsamplingPass, EffectPass, DepthOfFieldEffect, MaskFunction, DepthPickingPass, CopyPass, BlendFunction, BloomEffect, BrightnessContrastEffect, ChromaticAberrationEffect, ColorAverageEffect, ColorDepthEffect, DepthEffect, DotScreenEffect, GlitchEffect, GlitchMode, GodRaysEffect, GridEffect, HueSaturationEffect, NoiseEffect, OutlineEffect, PixelationEffect, 
ScanlineEffect, SelectiveBloomEffect, SepiaEffect, SSAOEffect, SMAAEffect, TextureEffect, ToneMappingEffect, VignetteEffect, ShockWaveEffect, LUT3DEffect, TiltShiftEffect: TiltShiftEffect$1, Effect, EffectAttribute } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:190:5) {
  digest: undefined
}
`

### Library versions used

 - Three: [^0.145.0]
 - @react-three/postprocessing: [^2.11.4]

### Desktop

 - OS: [Windows]
 - Browser [Chrome 113.0.5672.93]
 - Graphics hardware: [AMD Firepro]

**Comment by stphn** [comment]
Same here using:

`import { EffectComposer, Glitch } from '@react-three/postprocessing'`

While the server is running and the model is loaded:
Adding the following code **works as intended**. 

```javascript
<EffectComposer >
  <Glitch />
</EffectComposer>
```
But on **full page refresh** i receive the same following error as @mhmdnzr :

import pkg from 'postprocessing';
const { EffectComposer: EffectComposer$1, RenderPass, NormalPass, DepthDownsamplingPass, EffectPass, DepthOfFieldEffect, MaskFunction, DepthPickingPass, CopyPass, BlendFunction, BloomEffect, BrightnessContrastEffect, ChromaticAberrationEffect, ColorAverageEffect, ColorDepthEffect, DepthEffect, DotScreenEffect, GlitchEffect, GlitchMode, GodRaysEffect, GridEffect, HueSaturationEffect, NoiseEffect, OutlineEffect, PixelationEffect,
ScanlineEffect, SelectiveBloomEffect, SepiaEffect, SSAOEffect, SMAAEffect, TextureEffect, ToneMappingEffect, VignetteEffect, ShockWaveEffect, LUT3DEffect, TiltShiftEffect: TiltShiftEffect$1, Effect, EffectAttribute } = pkg;

Library versions used:

```
"@react-three/drei": "^9.68.3",
"@react-three/fiber": "^8.12.2",
"@react-three/postprocessing": "^2.11.4",
```

**Comment by vanruesc** [comment]
Should be fixed in `postprocessing@6.32.0`.
